CONTRIBUTING.md

```diff
@@ -95,6 +96,98 @@ have more control over the process and debug any issues.
 1. Run `rush change` to generate changelog. You will need to follow the interactive prompts.
    You can edit the added files later. If you don't add the right entries, the CI build will fail.
 
+# How to add and roll out new linter rules
+
+This section describes the process for adding a new rule to a ruleset. The rule will first be added to
+the staging pipeline, where it will not affect the ability to merge a PR. While the rule is in the
+staging pipeline, the author can verify that the rule is working correctly and not incorrectly marking
+violations. After verifying, the author can add the rule to the production pipeline so that the specs repo
+pipeline will block specs that violate the new rule from merging into any of the main/production branches.
+
+1. Ensure the new rule is set to run [only in staging](#how-to-set-a-spectral-rule-to-run-only-in-staging)
+1. Merge the new rule to the main branch. Once merged, your new rule will start running in the staging pipeline. You can
```

Comment:
Can you change the numbering accordingly? I see "1" for all the points.

Reply:
@tejaswiMinnu this a markdown feature: it is actually going to number it correctly once rendered.

Reply:
Markdown automatically puts the correct number when it renders the document.

E.g., if I put:
```markdown
1. a
1. b
1. c
```

It becomes:

1. a
1. b
1. c

I am following the style in the rest of this document here by using 1 for each number.

Reply:
I decided to switch it to 1, 2, 3 since it makes it easier to read it in plaintext as well 😄